base-server/index.test.ts

```diff
@@ -1175,17 +1175,26 @@ it('loads initial actions during subscription', async () => {
 it('calls unsubscribe() channel callback with logux/unsubscribe', async () => {
   let test = createReporter({})
   let client: any = {
-    node: { remoteSubprotocol: '0.0.0', onAdd: () => false }
+    node: {
+      remoteSubprotocol: '0.0.0',
+      remoteHeaders: { preservedSubscribedClientHeaders: true },
```

Comment:
```suggestion
      remoteHeaders: { preservedClientHeaders: true },
```

Let’s make them shorter